msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1120,6 +1121,27 @@ The `extensions.alternates` property is used to hide or prioritize specific in-m
+|`requirements.formFactors`| Array of enums | 2 | | Identifies the form factors that support the add-in.<br>Supported value: `desktop` |
+|`shortcuts`| Array | 20,000 | ✔️ | Specifies the custom keyboard shortcuts and the action each one performs. |
+|`shortcuts.key`| Object | | ✔️ | Specifies custom key combinations for a single action across supported platforms. |
+|`shortcuts.key.default`| String | 32 | ✔️ | Specifies the default key combination that can be used on all supported platforms if there isn't a specific combination configured for a particular platform. A keyboard shortcut can be any combination of the characters `A-Z`, `a-z`, `0-9`, and the punctuation marks `-`, `_`, and `+`. It must include the name of at least one modifier key (Alt, Ctrl, or Shift) and only one other key. These keys must be joined with a `+` character. For example, `Ctrl+Alt+1`. The The Shift key can't be used as the only modifier key. It must be combined with either the Alt or Ctrl key. On macOS, the Alt key is mapped to the Option key. On Windows, the Command key is mapped to the Ctrl key. |
+|`shortcuts.key.mac`| String | 32 | | Specifies the key combination of an add-in in Office on Mac. A keyboard shortcut can be any combination of the characters `A-Z`, `a-z`, `0-9`, and the punctuation marks `-`, `_`, and `+`. It must include the name of at least one modifier key (Alt, Command, or Shift) and only one other key. These keys must be joined with a `+` character. For example, `Command+1`. The Shift key can't be used as the only modifier key. It must be combined with either the Alt or Command key. On macOS, the Alt key is mapped to the Option key. |
+|`shortcuts.key.web`| String | 32 | | Specifies the key combination of an add-in in Office on the web. A keyboard shortcut can be any combination of the characters `A-Z`, `a-z`, `0-9`, and the punctuation marks `-`, `_`, and `+`. It must include the name of at least one modifier key (Alt/Option, Ctrl/Command, or Shift) and only one other key. These keys must be joined with a `+` character. For example, `Ctrl+Alt+1`. The Shift key can't be used as the only modifier key. It must be combined with either the Alt/Option or Ctrl/Command key.|
+|`shortcuts.key.windows`| String | 32 | | Specifies the key combination of an add-in in Office on Windows. A keyboard shortcut can be any combination of the characters `A-Z`, `a-z`, `0-9`, and the punctuation marks `-`, `_`, and `+`. It must include the name of at least one modifier key (Alt, Ctrl, or Shift) and only one other key. These keys must be joined with a `+` character. For example, `Ctrl+Alt+1`. The Shift key can't be used as the only modifier key. It must be combined with either the Alt or Ctrl key. |
+|`shortcuts.actionId`| String | 64 | ✔️ | Identifies the action that is performed when a keyboard shortcut is used. The `actionId` must match the `runtime.actions.id` property of an action in the `runtimes` object. |
```

Comment:
Please share schema for this feature if you have it.

Reply:
I shared a link to the file we referenced.